proposals/multi-cluster-config.md

```diff
@@ -0,0 +1,155 @@
+  dev_profile:
+  - aws_access_key_id     : **********
+  - aws_secret_access_key : **********
+  team_profile:
+  - ...
+```
+
+Other commands will have additional flags
+* **--cluster-config**
+* --cluster
+* --region
+* --aws-profile (this is --profile today)
+* **--ecs-profile**
+
+# Workflow
+### Single cluster first run
+1. Configure ECS credentials (Stores in `~/.ecs/ecs.profile`)
+   ```
+   ecs-cli configure  profile --access-key ********** --secret-key ********** --profile-name my_profile
+   ```
+1. Configure cluster (Stores in `~/.ecs/ecs.config`)
+   ```
+   ecs-cli configure --cluster cli-demo --region us-west-2 --config-name beta_config
+   ```
+1. Spin up a cluster
+   ```
+   ecs-cli up
+   ```
+1. Run tasks
+   ```
+   ecs-cli compose up
+   ```
+
+### Single cluster second run
+1. Configure another ECS profile
+   ```
+   ecs-cli configure  profile --access-key ********** --secret-key ********** --profile-name gamma_profile
+   ```
+1. Set default ECS profile
+   ```
+   ecs-cli configure profile default --profile-name gamma_profile
+   ```
+1. Configure another cluster
+   ```
+   ecs-cli configure --cluster cli-demo --region us-west-2 --config-name gamma_config
+   ```
+1. Set default cluster
+   ```
+   ecs-cli configure default --config-name gamma_config
+   ```
+1. Spin up a cluster
+   ```
+   ecs-cli up
+   ```
+1. Run tasks
+   ```
+   ecs-cli compose up
+   ```
+
+### Multiple clusters
+1. Configure ECS profile
+   ```
+   ecs-cli configure  profile --access-key ********** --secret-key ********** --profile-name my_profile
+   ```
+1. Configure and spin up the first cluster
+   ```
+   ecs-cli configure --cluster cli-demo --region us-west-2 --config-name beta_config
+   
+   ecs-cli up --cluster-config beta_config
+   ```
+1. Configure and spin up a second cluster
+   ```
+   ecs-cli configure --cluster cli-demo2 --region us-west-2 --config-name prod_config
+      
+   ecs-cli up --cluster-config prod_config
+   ```
+1. Run tasks on different clusters
+   ```
+   ecs-cli compose up --cluster-config beta_config
+
+   ecs-cli compose up --cluster-config beta_config
+   ```
+1. Do the same on a different account
+   ```
+   ecs-cli configure  profile --access-key ********** --secret-key ********** --profile-name prod_profile
+
+   ecs-cli up --cluster-config prod_config --ecs-profile prod_profile
+
+   ecs-cli compose up --cluster-config prod_config --ecs-profile prod_profile
+   ```
+1. Do the same with AWS profile
+   ```
+   ecs-cli up --cluster-config prod_config --aws-profile default
+
+   ecs-cli compose up --cluster-config prod_config --aws-profile default
+   ```
+
+### Ad-hoc
+Does not need to configure cluster
+1. Configure ECS profile
+   ```
+   ecs-cli configure  profile --access-key ********** --secret-key ********** --profile-name my_profile
+   ```
+1. Cluster PS
+   ```
+   ecs-cli ps --cluster cli-demo --region us-west-2 --ecs-profile my_profile
+   ```
+1. Do the same with AWS profile
```

Comment:
Isn't it redundant to specify --region when it's already in the profile?  Why not just use --profile like every other AWS command?